### PR TITLE
Cleanup README formatting

### PR DESCRIPTION
**Pull request by namoscato**
Thanks for the great documentation; this was a breeze to get running locally!

* Adds `.nvmrc`
* Updates README formatting (mostly ordered list indentation so feel free to ignore whitespace on review)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,77 +12,72 @@ Fishbowl is built with [Material UI](https://material-ui.com/), [Typescript](htt
...
+    ```bash
+    brew bundle
+    ```
 
-`nvm` is used to specify the node version it requires, based on `.nvmrc` file.
+3. Install Node.js via [Node Version Manager](https://github.com/nvm-sh/nvm):
 
-```bash
-nvm install
-nvm use
```
Comment (namoscato) [inline README.md:33]: `nvm install` will inherently use the installed version.

File: README.md [hunk 2]
```diff
@@ -12,77 +12,72 @@ Fishbowl is built with [Material UI](https://material-ui.com/), [Typescript](htt
...
 
-See more detailed instructions [here](https://hasura.io/docs/1.0/graphql/manual/hasura-cli/install-hasura-cli.html)
+    [Hasura](https://hasura.io/) is a GraphQL server that gives you instant, realtime GraphQL APIs over Postgres, with webhook triggers on database events, and remote schemas for business logic.
 
-6. Install Hasura CLI
+    ```bash
+    curl -L https://github.com/hasura/graphql-engine/raw/stable/cli/get.sh | bash
+    ```
 
-See https://docs.hasura.io/1.0/graphql/manual/hasura-cli/install-hasura-cli.html
```
Comment (namoscato) [inline README.md:52]: This seemed to be describing the same thing as above, but let me know if I'm missing something.

File: .nvmrc [hunk 3]
```diff
@@ -0,0 +1 @@
+12.16.3
```
Comment (namoscato) [inline .nvmrc:1]: Ah, I actually just found this in `app/`. Let me know what you want me to do here, i.e. happy to update the instructions to `cd app`.

File: README.md (hunk 2 above)
Comment (avimoondra) [reply README.md:52]: in case it changes, but 🤷 keep it short :)

File: .nvmrc (hunk 3 above)
Comment (avimoondra) [reply .nvmrc:1]: actually I'll delete the one in app/ !

now there is the actions-server subdir (it's a node express server), we should probably use the same version of node across both :)

**Pull request merged**